src/Input.js

```diff
@@ -6,23 +6,36 @@ import uuid from 'node-uuid';
 
 class Input extends React.Component {
   componentDidMount() {
-    if (this.props.type === 'select' && !this.props.browserDefault && typeof $ !== 'undefined') {
-      $(ReactDOM.findDOMNode(this.refs.inputEl)).material_select();
-    }
+    this._setupEl();
   }
-
   componentDidUpdate() {
-    if (this.props.type === 'select' && !this.props.browserDefault && typeof $ !== 'undefined') {
-      var $el = $(ReactDOM.findDOMNode(this.refs.inputEl));
-      $el.material_select('destroy');
-      $el.material_select();
+    this._setupEl();
+  }
+
+  _setupEl() {
+    if (typeof $ !== 'undefined') {
+      var $inputEl = $(ReactDOM.findDOMNode(this.refs.inputEl));
```

Comment:
`ReactDOM.findDOMNode` is not necessary, the ref is the DOM now. see https://facebook.github.io/react/blog/2015/10/07/react-v0.14.html#dom-node-refs